doc/man3/PEM_read_bio_PrivateKey.pod

```diff
@@ -78,7 +91,11 @@ PEM_write_bio_PKCS7, PEM_write_PKCS7 - PEM routines
                               OSSL_LIB_CTX *libctx, const char *propq);
  EVP_PKEY *PEM_read_PUBKEY(FILE *fp, EVP_PKEY **x,
                            pem_password_cb *cb, void *u);
+ int PEM_write_bio_PUBKEY_ex(BIO *bp, EVP_PKEY *x,
```

Comment:
Maybe describe the libctx, propq args in the
 PEM FUNCTION ARGUMENTS section?

Reply:
There already is a paragraph at the end of that section.

Reply:
Yeah it is about 2 pages of content down :)